src/index.js

```diff
@@ -6,3 +14,13 @@ export * from './mixins/'
 export * from './lib/'
 export * from './lib/validation/'
 
+
+export default {
+  ...shared,
+  ...directives,
+  ...utils,
+  ...components,
+  ...mixins,
+  ...lib,
+  ...validation
+}
```

Comment:
I don't understand why, but for some reason we need the default export AND the named Exports. I would love to understand that. I think then it should be easy to resolve it

Reply:
I've come across the same issues/fix within the [alias draft PR](https://github.com/demos-europe/demosplan-ui/pull/324). As i've seen the same pattern (having default exports besides named exports): maybe it's ok to do so.

Reply:
Here: https://github.com/demos-europe/demosplan-ui/pull/324/files#diff-6607fbfb793b10562651564bfa45bd5b8be08e5456b9a0b085341c8984a44e06R84-R89